### ISSUE TITLE
Feature Request: Add method to clear upload state

### ISSUE THREAD

**Issue by 8807p3** [tanaikech/GeminiWithFiles#5]
Hi @tanaikech, Thank you for creating this awesome GeminiWithFiles library. It's very useful for handling file uploads to Gemini API.

## Current Behavior
When using the resumable upload feature, if an upload fails or is interrupted, the upload state is stored in ScriptProperties. However, there's currently no way to clear this state from the main script that's using the library.

## Issue
If we need to start a new upload after a failed upload, we can't clear the previous upload state. This causes the library to continue trying to upload the old file instead of starting fresh with the new file.

## Proposed Solution
Could you please add one or more of these features:

1. Add a method to clear the upload state:
```javascript
// Static method
GeminiWithFiles.clearUploadState();

// or instance method
const gemini = new GeminiWithFiles({ apiKey: "..." });
gemini.clearUploadState();

**Comment by tanaikech** [comment]
Thank you for your comment. I implemented it as v2.0.3. https://github.com/tanaikech/GeminiWithFiles?tab=readme-ov-file#v203

Also, from v2.0.3, when you use `setFileIdsOrUrlsWithResumableUpload`, please include `propertiesService: PropertiesService.getScriptProperties()` into the initial object as follows. Because, when `PropertiesService.getScriptProperties()` is used in the library, the values are put into the library. When I created [Ref](https://github.com/tanaikech/UploadApp) and [Ref](https://medium.com/google-cloud/uploading-large-files-to-gemini-with-google-apps-script-overcoming-50-mb-limit-6ea63204ee81), I supposed that the script is used by copying and pasting instead of the library. So, I included `PropertiesService.getScriptProperties()` in the script. But I noticed that when this is used with GeminiWithFiles, each user is required to use `PropertiesService.getScriptProperties()`. So, I modified this. Please be careful about this.

**Comment by tanaikech** [comment]
Thank you for replying. I guessed that there might be a situation in which another data might be required to be uploaded after an error occurs. So, in the current stage, in order to clear the resumable upload status, I would like to use the above option. And, if an error occurs and it is required to upload the same data again, the users can achieve it by clearing the status and running the script again.
